### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by kazup01**
screen shot 0029-07-10 at 5 51 01 pm
screen shot 0029-07-10 at 5 51 12 pm

### CODE REVIEW

File: browser/main/Detail/SnippetNoteDetail.js [hunk 1]
```diff
@@ -433,10 +436,18 @@ class SnippetNoteDetail extends React.Component {
...
+    if (infoPanel.style) infoPanel.style.display = infoPanel.style.display === 'none' ? 'inline' : 'none'
+  }
+
   render () {
-    let { data, config } = this.props
+    let { data, config, location } = this.props
     let { note } = this.state
 
+    let storageKey = note.storage
+    let folderKey = note.folder
```
Comment (asmsuechan) [inline browser/main/Detail/SnippetNoteDetail.js]: You should use `const` instead of `let`.

File: browser/main/Detail/SnippetNoteDetail.js [hunk 2]
```diff
@@ -491,6 +502,17 @@ class SnippetNoteDetail extends React.Component {
...
+    let options = []
+    data.storageMap.forEach((storage, index) => {
+      storage.folders.forEach((folder) => {
+        options.push({
+          storage: storage,
+          folder: folder
+        })
+      })
+    })
+    let currentOption = options.filter((option) => option.storage.key === storageKey && option.folder.key === folderKey)[0]
```
Comment (asmsuechan) [inline browser/main/Detail/SnippetNoteDetail.js]: You should use `const` instead of `let`.

File: browser/main/Detail/SnippetNoteDetail.js (hunk 1 above)
Comment (kazup01) [inline browser/main/Detail/SnippetNoteDetail.js]: Done💯

File: browser/main/Detail/SnippetNoteDetail.js (hunk 2 above)
Comment (kazup01) [inline browser/main/Detail/SnippetNoteDetail.js]: Doen🎶

**Pull request merged**